### DISCUSSION TITLE
Remove group tabs

### DISCUSSION THREAD

**Discussion by kaporli** [mbnuqw/sidebery#642] [Q&A]
Is it possible to not display the tab containing my folder and have the folder ONLY show in the tab tree?
<img width="834" alt="image" src="https://user-images.githubusercontent.com/52631424/170879972-2b890086-54c4-489c-a506-1716f0397fbc.png">

<img width="273" alt="image" src="https://user-images.githubusercontent.com/52631424/170880015-b0d5f90f-63a2-4888-9d16-772a6eced17e.png">

**Comment by mbnuqw** [comment]
No, it's not possible right now. You can try to use panels for this: panel for work, panel for whatever...etc.
But I can add an option to hide such tabs (in unfolded state) from the native tabs strip in v5.
